src/pom.xml

```diff
@@ -135,8 +135,9 @@
         <sparkbench.mr.version>2</sparkbench.mr.version>
       </properties>
       <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
With this change, does people who wants to build again MR1 has to deactivate MR2 using !MR2? If so, we'd better not activate any profiles by default.@mwws